### PR TITLE
issue 129 - Windows support

### PR DESCRIPTION
**Pull request by rossoha**
move rm calls to Java/Scala API

### CODE REVIEW

**Comment by oyvindberg** [comment]
Thank you!

I have two remarks:
- there is already some file handling stuff in [files.scala](https://github.com/ScalablyTyped/Converter/blob/master/utils/src/main/scala/org/scalablytyped/converter/internal/files.scala). Can you please write a method `deleteAll` there which takes an `os.Path` and uses `os.remove.all` on windows and `rm` on other platforms? The performance cost is real because the CI job does this 8500 times, so a second wasted is 8500 seconds added together :)
Just remove logging of these removals, it's not important.

- Secondly, I think we should adapt [this directory](https://github.com/ScalablyTyped/Converter/blob/master/utils/src/main/scala/org/scalablytyped/converter/internal/constants.scala#L11) for windows users, `~/.cache` is foreign there. Can you see if you find some logic to use `~/Library/Cache` (or whatever it is) on mac, `~/.cache` for linux and another directory for windows? If I'm not mistaken there are different levels of such directories there: We want one that is persisted across restarts, but not moved along with a roaming users files if that makes sense?

**Comment by oyvindberg** [comment]
- Where is `~/.ivy2` on windows? Might have to update that as well

**Comment by rossoha** [comment]
By default, sbt uses the standard Ivy home directory location `${user.home}/.ivy2/`
https://www.scala-sbt.org/1.x/docs/Library-Management.html

**Comment by rossoha** [comment]
@oyvindberg  Please take a look.

**Comment by sorenbug** [comment]
Caches and etc are usually stored in Local AppData on Windows:

image

File: importer/src/main/scala/org/scalablytyped/converter/internal/importer/Ci.scala [hunk 1]
```diff
@@ -195,7 +195,7 @@ class Ci(config: Ci.Config, paths: Ci.Paths) {
           Try(interfaceCmd.runVerbose git ("clean", "-fdX")) // remove ignored files/folders
           Try(interfaceCmd.runVerbose git ("clean", "-fd"))
           Try(interfaceCmd.runVerbose git ('reset, "--hard", "origin/master"))
-          Try(interfaceCmd.runVerbose rm ("-f", ".git/gc.log"))
+          Try(files.delete(projectFolder / ".git/gc.log"))
```
[the remark this answers is not in the document]
Comment (rossoha) [reply importer/src/main/scala/org/scalablytyped/converter/internal/importer/Ci.scala]: Done

**Comment by rossoha** [comment]
@oyvindberg Please take a look.

**Comment by oyvindberg** [comment]
Looks good, I'll try to get time to merge and release it tonight

**Pull request merged**